perf/src/sigverify.rs

```diff
@@ -21,7 +20,11 @@ use {
         hash::Hash, message::MESSAGE_HEADER_LENGTH, pubkey::Pubkey, short_vec::decode_shortu16_len,
         signature::Signature,
     },
-    std::sync::atomic::{AtomicU64, Ordering},
+    std::collections::hash_map::RandomState,
+    std::hash::BuildHasher,
+    std::hash::Hasher,
```

Comment:
I think you're getting siphasher here actually. wouldn't it be better to specify ahash explicitly so we know what we're getting?

Reply:
switched to ahash